en/animation/joint-texture-layout.md

```diff
@@ -0,0 +1,88 @@
+
+![joint texture layout panel](./joint-texture-layout/joint-texture-layout-panel.png)
+
+> **Note**: the joint texture layout settings panel provides, essentially, runtime **memory allocation guidance rules**. For specified skeleton and animation assets, they are guaranteed to be allocated according to the specified rules. However, if an asset is used at runtime that is not specified by the rules, it will go back to the automatic allocation mode of global reuse.
+
+## Joint texture layout settings
+
+Here we use the **instanced-skinning** scenario from the example project [show-cases](https://github.com/cocos-creator/example-3d/tree/v3.0/show-cases/assets/scenes) as an example to see how the joint texture layout is set up and how it works. The following figure shows a sample scene.
+
+The following figure shows a sample scenario with multiple instances from the same model, playing completely different animations at the same time. As you can see, the current scene, plus the UI, has a total Drawcall of 60 and an instance count of 0. This state will be used as the basis for later changes to compare.
+
+![Baseline](./joint-texture-layout/instancing_baseline.gif)
+
+To create a model with instancing version turned on, the following steps are required:
+
+1. In the **Inspector** panel, check the `UseBakedAnimation` property in the SkeletalAnimation component (using pre-baked animation mode).
+
+    ![use baked animation](./joint-texture-layout/use-baked-animation.png)
+
+2. check `USE INSTANCING` for all materials used by SkinningModel.
+
+    ![Enabling](./joint-texture-layout/enabling_instancing.png)
+
+In our example scene we actually made two sets of Prefab and set the material diffuse color to blue in the instancing version in order to see and distinguish the performance of both systems. You can see that the effect has been exactly correct, and that only 5 Drawcalls (each model is divided into 5 parts) were used, with 45 instances.
+
+![Normal](./joint-texture-layout/instancing_normal.gif)
+
+> **Note**: the reason for rendering all models correctly here is that the amount of animation data is still relatively small, and the generic joint texture global reuse logic already writes all animation data to the same texture, so the effect is correct. However, if the size of the default joint texture (360 * 360) is exceeded by new animations that may be added at any time, the animation will definitely go wrong, which is why the joint texture layout panel must exist.
+
+## Joint Texture Layout Effects
+
+For demonstration purposes, we can intentionally put each animation on a separate texture in the **Joint Texture Layout Settings** panel to see the final rendering effect.
+
+First, open the **Joint Texture Layout Settings** panel:
+
+![Panel](./joint-texture-layout/joint_texture_layout_new.png)
+
+The three **+** in the panel are used as follows:
+
+- ① - Used to add Texture units, a Texture unit consists of multiple Skeleton units.
```

Comment:
```suggestion
- Used to add Texture units, a Texture unit consists of multiple Skeleton units.
```

Reply:
![image](https://user-images.githubusercontent.com/55615178/129340859-82f82c17-f94c-4414-81ac-b5b8433e459d.png)
@slackmoehrle These numbers can't be deleted, because they corresponds to the numbers in this figure.

Reply:
Oh, I get it. Can we please make them normal numbers and not a styled font....i.e 1,2,3